aviary/mission/flops_based/phases/detailed_landing_phases.py

```diff
@@ -37,10 +37,7 @@
     InitialGuessState, PhaseBuilderBase)
 from aviary.utils.aviary_values import AviaryValues
 from aviary.variable_info.functions import setup_trajectory_params
-from aviary.variable_info.variables import Dynamic as _Dynamic
-from aviary.variable_info.variables import Mission
-
-Dynamic = _Dynamic.Mission
+from aviary.variable_info.variables import Dynamic, Mission
```

Comment:
All of these detailed landing phases have a "max_range" which should become "max_distance".

Reply:
John was hoping to name distance_max as range I think. 
we should never be using a naming scheme with `max` at the start.